### ISSUE TITLE
index.d.ts and index.js don't seem to be in sync

### ISSUE THREAD

**Issue by tbelch-at-eHealth-Tec** [paragonie/ciphersweet-js#21]
Hi!

I see an exported member `Transformation` in [index.js](https://github.com/paragonie/ciphersweet-js/blob/ff9052c02ed8393bcd7493d138cf32ffe75bcab9/index.js#L23). But I can't find the TypeScript type. The only thing, which would fit is the type `Transform` in [index.d.ts](https://github.com/paragonie/ciphersweet-js/blob/ff9052c02ed8393bcd7493d138cf32ffe75bcab9/index.d.ts#L256).

But using `Transform` leads to the following error:

```
TypeError: Class extends value undefined is not a constructor or null

      1 | import { Transform } from "ciphersweet-js";
      2 |
    > 3 | export class FirstNCharacters extends Transform {
        |                                       ^
      4 |   private n: number;
      5 |
      6 |   constructor(n: number) {
```

And using `Transformation`, I get the following TypeScript error:

```
Module '"ciphersweet-js"' has no exported member 'Transformation'.ts(2305)
```

How comes? Could that be fixed soon?

Best regards,
Tobias

**Comment by tbelch-at-eHealth-Tec** [comment]
I found a workaround. I created a `js` file which requires `Transformation` and exports it. I can import that module in my TypeScript code now. But a real fix for this would be most welcomed.

```
transformation.ciphersweet.js

const { Transformation } = require("ciphersweet-js");
export default Transformation;

```

```
// in a TypeScript file
import Transformation from "./transformation.ciphersweet";
```

**Comment by paragonie-security** [comment]
This should be fixed in `v2.0.4`.
